### PR TITLE
Net Investment Income Tax

### PR DESCRIPTION
**Pull request by Amy-Xu**
This Net Income Income Tax (NIIT), also referred as Unearned Income Medicare Contribution Tax, was originally added to c05750 (Tentative income tax before credits) in MUI() but not the final tax. Right now moved it from c05750 to _othertax to make sure it's included in the final _ospctax.

### CODE REVIEW

File: taxcalc/functions.py [hunk 1]
```diff
@@ -786,6 +774,13 @@ def AMTI(  c60000, _exact, e60290, _posagi, e07300, x60260, c24517,
               _amt25pc, c62747, c62755, c62770, _amt, c62800,
               c09600, _othtax, c05800)    
 
+@iterate_jit(parameters=["_NIIT_thd", "NIIT_trt"], nopython=True)
+def MUI(c00100, _NIIT_thd, MARS, c05750, e00300, e00600, c01000, e02000, NIIT_trt, NIIT):
```
Comment (MattHJensen) [inline taxcalc/functions.py]: Do we still need c05750 here?

File: taxcalc/params.json [hunk 2]
```diff
@@ -1131,8 +1131,174 @@
...
+        "start_year": 2013,
+        "col_var": "NA",
+        "row_var": "NA",
+        "row_label": "NA",
+        "cpi_inflated": false,
+        
+        "col_label": "NA",
+        "value": [0]
+    },
+        "_CG_rt3":{
```
Comment (MattHJensen) [inline taxcalc/params.json]: Do we need _CG_rt3? Threshold 3 (and 2?)? Same with Dividends?

**Comment by Amy-Xu** [comment]
Just removed c05750 from the MUI function signature. I don't think we'll need CG_thd3, but we do need three rates for capital gain under current law. Not really sure about the dividend part, will look into it.

**Comment by MattHJensen** [comment]
Excellent. Merging!

**Pull request merged**